docs/source/frameworks/onnx.rst

```diff
@@ -6,22 +6,10 @@ ONNX
 Preface
 -------
 
-In this tutorial, we will show how to train a model with PyTorch
-TensorFlow and Scikit Learn, then we will convert resulting model to
-ONNX format, use BentoML to save it and create a prediction service.
-
-.. note::
-
-   BentoML currently only support `ONNX Runtime
-   <https://onnxruntime.ai>`_ as ONNX backend. BentoML requires either
-   ``onnxruntime>=1.9`` or ``onnxruntime-gpu>=1.9`` to be installed.
-
-.. seealso::
-
-   ONNX support multiple machine learning frameworks and each
-   framework may have unique requirements/recommendations for
-   converting its own model format to ONNX format. Please refer to
-   resources listed below for in-depth knowledge of each framework.
+ONNX support multiple machine learning frameworks and each framework
+may have unique requirements/recommendations for converting its own
```

Comment:
```suggestion
ONNX is an open format built to represent machine learning models. ONNX provides `high interoperability <https://onnx.ai/supported-tools.html#buildModel>`_  among various frameworks, as well as enable machine learning practitioners to maximize models' performance across `different hardware <https://onnx.ai/supported-tools.html#deployModel>`_. 

Due to its high interoperability among frameworks, we recommend you to check out the framework integration with ONNX as it will contain specific recommendation and requirements for that given framework.

```